server/src/ast/mod.rs

```diff
@@ -34,15 +40,262 @@ pub enum SyntaxTree {
 }
 
 // TODO check whether necessary
-// pub enum Values {
-//     Variable(String),
-//     Number(BigDecimal),
-//     Sum(Vec<Values>),
-//     Product(Vec<Values>),
-//     Exponent(Vec<Values>),
-//     AddInv(Box<Values>),
-//     MulInv(Box<Values>),
-// }
+
+#[derive(Clone, Debug, Eq, PartialEq, Hash, Ord, PartialOrd)]
+pub enum Values {
```

Comment:
possible introduce a reference to LocalizedSyntaxNode/Values and indicating how it got introduced